pegasus/sites.v3/code.org/public/teacher-dashboard/review-hociyskvuwa/splat.haml

```diff
@@ -86,5 +98,7 @@ title: Review Hour of Code special events
       type: "post",
       contentType: "application/json; charset=utf-8",
       data: JSON.stringify( { review: review } )
+    }).fail(function() {
```

Comment:
Since we're showing an error message, do you think it would be useful to also show a success message?

    $.ajax({
     ...
     }).done(processSuccess).fail(processError);